readme.md

```diff
@@ -232,6 +232,13 @@ Content about: MongoDB, SQL, C#, C++ \
 Featured playlists: `MongoDB - NoSQL Database`, `PHP - Programming Language`, `C# - Programming Language`, `C++ - Programming Language`. \
 <br/>
 
+[<img align="left" height="94px" width="94px" alt="The Cherno Project's avatar" src="https://yt3.ggpht.com/a/AATXAJwnC9ATEyVr8Mf-Vz3K1hAUMzPXjxz_rp2LZL6JVg=s100-c-k-c0xffffffff-no-rj-mo"/>](https://www.youtube.com/user/TheChernoProject)
+
+[**The Cherno**](https://www.youtube.com/user/TheChernoProject) \
```

Comment:
The channel is a verified YouTube channel and it uploads videos every week. It should have those two [badges](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/credits.md).